examples/execsnoop.bpf.c

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
```

Comment:
Where exactly is this from?

Reply:
From [bootstrap](https://github.com/libbpf/libbpf-bootstrap#bootstrap) in libbpf-bootstrap, and [execsnoop](https://github.com/iovisor/bcc/blob/master/libbpf-tools/execsnoop.bpf.c) from bcc/libbpf-tools.

This header declare is from https://github.com/libbpf/libbpf-bootstrap/blob/master/examples/c/bootstrap.bpf.c.